src/modules/accounts/dto/accounts.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { IsNotEmpty, MinLength, IsEmail, IsEnum } from 'class-validator';
+
+export class AccountDto {
+    @IsNotEmpty()
+    readonly name: string;
+    readonly createAt: Date;
+    readonly updateAt: Date;
+    readonly deleteAt: Date;
```

Comment:
the sequelize model does not have createdAt and updatedAt.
We are passing these values directly to account model here - https://github.com/plant99/nestjs-playground/blob/1906d7fa8942ac97093a5b91a1e4da2ec943a486/src/modules/auth/auth.service.ts#L34-L35